diff -rN -u old-hlint/src/Util.hs new-hlint/src/Util.hs
--- old-hlint/src/Util.hs	2013-06-27 13:19:41.288856463 +0000
+++ new-hlint/src/Util.hs	2013-06-27 13:19:41.304855279 +0000
@@ -70,8 +70,10 @@
 headDef x [] = x
 headDef x (y:ys) = y
 
+#if !MIN_VERSION_base(4,7,0)
 isLeft Left{} = True; isLeft _ = False
 isRight = not . isLeft
+#endif
 
 unzipEither :: [Either a b] -> ([a], [b])
 unzipEither (x:xs) = case x of
